docs/license.md

```diff
@@ -0,0 +1,22 @@
+[Full license](https://github.com/triplea-game/triplea/blob/master/LICENSE)
```

Comment:
Probably dangerous to call the target document the "full license," as the notice below encapsulates our complete license agreement due to the exception we had to grant for non-GPL-compliant libraries.  If you want to provide this link, I would recommend simply hyperlinking the phrase `GNU General Public License` on line 12 below.  However, the _LICENSE_ file is identical to what's available on `gnu.org` (a link to which also appears below), so it's probably not necessary to link it twice.

Reply:
Backing up, do you know if we need a LICENSE file at all? Could we just link to the `gnu.org` copy?

Reply:
Primarily, _LICENSE_ permits GitHub to deduce the repo's license for display:

![license](https://user-images.githubusercontent.com/4826349/50739286-e9bbf580-11ab-11e9-8b39-d0f8dbea1159.png)

Secondarily, I guess, it's just convention for OSS projects to have a canonical location in which to find the project's license.  The [GitHub docs](https://help.github.com/articles/licensing-a-repository/#determining-the-location-of-your-license) simply state (emphasis mine):

> Most people place their license text in a file named LICENSE.txt (or LICENSE.md) in the root of the repository; here's an example from Hubot.
>
> Some projects include information about their license in their README. For example, a project's README may include a note saying "This project is licensed under the terms of the MIT license."
>
> **As a best practice, we encourage you to include the license file with your project.**

Reply:
In the meantime, I'm happy to update the link text to not say "full license", done in: b7164b4fc

Reply:
> I'm wondering if we could simply add our additional grants to the top of the existing LICENSE file

I believe I tried that when we upgraded to GPLv3.  IIRC, GitHub could no longer deduce our license, as evidenced by the red box I illustrated above being empty.